source/braille.py

```diff
@@ -2028,7 +2034,7 @@ def message(self, text):
 		If a key is pressed the message will be dismissed by the next text being written to the display.
 		@postcondition: The message is displayed.
 		"""
-		if not self.enabled or config.conf["braille"]["messageTimeout"] == 0 or text is None:
+		if not self.enabled or config.conf["braille"]["showMessages"] == ShowMessages.DISABLED or text is None:

```

Comment:
new lines for readability

```suggestion
		if (
			not self.enabled
			or config.conf["braille"]["showMessages"] == ShowMessages.DISABLED
			or text is None
		):
```